packages/create-hint/package.json

```diff
@@ -9,6 +9,10 @@
   "bin": {
     "create-hint": "./dist/src/create-hint.js"
   },
+  "dependencies": {
+    "@hint/utils": "^1.0.0",
```

Comment:
All of these should be `devDependencies`, we bundle everything with webpack and we don't want to download unnecessary dependencies when doing a `npm create hint`